### ISSUE TITLE
Dualsense raw input/disable xinput emulation

### ISSUE THREAD

**Issue by Hadrianneue** [Frogging-Family/wine-tkg-git#938]
When playing a game on steam that supports the dualsense controller natively, like F1 2021, i have to disable steam input so the game can see the controller "raw" dinput interface (i don't actually know how it works, but that is my guess), otherwise the game just picks the steam input interface...

On lutris, by either using wine-ge or lutris-ge, by default it seems, either it exposes both dinput/xinput or just dinput, making it possible for games that support the dualsense natively like metro exodus enhanced edition to properly see the raw dualsense interface and use it properly...

is there a configuration to build wine-tkg to make it behave ike wine-ge regarding controller support? it appear that wine-tkg is not exposing the dualsense in a way that allows games that uses it natively to actually use it properly, like wine-ge does on lutris or proton-ge does on steam (with steam input disabled).

i've already tried disabling lutris runtime/prefer system libraries or overriding the xinput controller on wine's control panel or  replugging the controller...

**Comment by gabriele2000** [comment]
I also have a similar question: what does the xinput override as dinput do?
Like, I noticed that every game see my dualsense as an xinput controller.
If I override it as dinput, many games stop working.
That bothers me because I think that I'm missing out on features... correct me if I'm wrong.

**Comment by Hadrianneue** [comment]
> If I override it as dinput, many games stop working.

yeah wine-ge is able to expose the dualsense controller because it doesn't seem to enforce xinput with lutris, in fact there is a patch that works on some games to provide haptic feedback, someone made an easy to use patch github.com/ValveSoftware/Proton/issues/5900 (read Mutcholoko comment at the end, he provides a pre-compiled build to be used with steam), would be nice if that is also added to wine-tkg

**Comment by Hadrianneue** [comment]
building without SDL patches didn't work, in fact it says on non-makepkg-build.sh that sdl_joy_support="true" but i couldn't find it (it set to false on every instance by default) i did comment the proton-sdl-joy source on prepare.sh, didn't make any difference while compiling...

building with wine-tkg-valve-exp-bleeding did work just like wine-ge, but i doesn't allow some configurations like building with _NOMINGW="true" unfortunately...
